demos/src/hover.mustache

```diff
@@ -0,0 +1,17 @@
+<div class="o-colors-page-background">
+	<div class='demo-tooltip-container' id="box-tooltip">
+
+		<button class='o-buttons o-buttons--big demo-tooltip-target' id="demo-tooltip-target">
+			Share
+		</div>
+
+		<div data-o-component="o-tooltip"
+		data-o-tooltip-position="right"
+		data-o-tooltip-target="demo-tooltip-target"
+		data-o-tooltip-show-on-hover="true">
```

Comment:
If this is not quoted I think it will return a boolean instead of a string, which may be useful when it comes to https://github.com/Financial-Times/o-tooltip/pull/10/files#diff-12b4ef0bf4145d4278dc13ca533b5197R75